src/util/system.cpp

```diff
@@ -890,25 +773,31 @@ bool ArgsManager::ReadConfigFiles(std::string& error, bool ignore_invalid_keys)
         bool emptyIncludeConf;
         {
             LOCK(cs_args);
-            emptyIncludeConf = m_override_args.count("-includeconf") == 0;
+            std::vector<util::SettingsValue>* includes = util::FindKey(m_settings.command_line_options, "includeconf");
```

Comment:
I believe it could be **const** std::vector<util::SettingsValue>* **const** includes. Neither value (vector) nor pointer is ever changed (right?).